pandas/tests/extension/test_categorical.py

```diff
@@ -153,7 +153,7 @@ def test_arith_frame_with_scalar(self, data, all_arithmetic_operators, request):
                     reason="rmod never called when string is first argument"
                 )
             )
-        super().test_arith_frame_with_scalar(data, op_name)
+        super().test_arith_frame_with_scalar(data, op_name, request)
```

Comment:
The request in this file don't look like they are used?

Reply:
(and same generally throughout this PR)?

Reply:
I've just pushed some changes to use LSP (Linkov Substitution Principle)-style, which may strike you as more Pythonic.  It definitely cleans up the cases that offended, at a small cost of using `*args, **kwargs` instead of `request`.